torch_em/model/unetr.py

```diff
@@ -0,0 +1,299 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from typing import Tuple
+from functools import partial
+from torch_em.model.unet import Decoder, ConvBlock2d, Upsampler2d
+
+from segment_anything.modeling import ImageEncoderViT
+
+from micro_sam.util import get_sam_model
+
+
+# This class (inherits from Segment Antything) and its supporting functions below lightly adapted from the ViTDet backbone available at: https://github.com/facebookresearch/detectron2/blob/main/detectron2/modeling/backbone/vit.py # noqa
```

Comment:
This comment is a bit confusing. Say that you're inheriting from the SAM ViT and what exactly you're changing.